src/main/java/cloud/fogbow/ras/core/plugins/interoperability/opennebula/compute/v5_4/OpenNebulaComputePlugin.java

```diff
@@ -30,17 +30,20 @@
 import cloud.fogbow.ras.core.plugins.interoperability.ComputePlugin;
 import cloud.fogbow.ras.core.plugins.interoperability.util.LaunchCommandGenerator;
 
+import javax.annotation.Nullable;
+
 public class OpenNebulaComputePlugin implements ComputePlugin<CloudUser> {
 
 	private static final Logger LOGGER = Logger.getLogger(OpenNebulaComputePlugin.class);
 
 	private static final int MB_CONVERT = 1024;
+	protected static final int DEFAULT_NUMBER_OF_INSTANCES = 1;
 
-	private static final String DEFAULT_ARCHITECTURE = "x86_64";
-	private static final String DEFAULT_GRAPHIC_ADDRESS = "0.0.0.0";
-	private static final String DEFAULT_GRAPHIC_TYPE = "vnc";
-	private static final String NETWORK_CONFIRMATION_CONTEXT = "YES";
-	private static final String NIC_IP_EXPRESSION = "//NIC/IP";
+	protected static final String DEFAULT_ARCHITECTURE = "x86_64";
```

Comment:
I imagine you've changed the visibility of these variables to be able to use them in tests. If that's the case, maybe we should consider to have them in testUtils class, once they can be used in another plugin's tests, does it makes sense?

Reply:
yes, that's for testing. all of these constants are compute related, so i don't see other plugin tests using them. but i can sure move them if you really see fit. ;-)